doc/docs/index.md

```diff
@@ -2,6 +2,8 @@
 # MPB
 ---
 
+Howdage!
```

Comment:
My mistake, now removed. I was trying to make random changes in an effort to figure out why the latest doc tree didn't seem to be reflected on ReadTheDocs, but it turned out I needed to clear my browser cache.